libcnb-test/tests/integration_test.rs

```diff
@@ -132,14 +133,29 @@ fn starting_containers() {
 
 #[test]
 #[ignore = "integration test"]
-#[should_panic(
-    expected = "Could not package current crate as buildpack: BuildBinariesError(ConfigError(NoBinTargetsFound))"
-)]
 fn buildpack_packaging_failure() {
-    TestRunner::default().build(
-        BuildConfig::new("libcnb/invalid-builder", "test-fixtures/empty"),
-        |_| {},
-    );
+    let crate_dir = env::var("CARGO_MANIFEST_DIR").unwrap();
+    let result = std::panic::catch_unwind(|| {
```

Comment:
[Comment, no action] TIL `std::panic::catch_unwind` it seems like this gives us more control in the test. Neat.